packages/config/config/devices/0x017f/wnk-mot1.json

```diff
@@ -60,19 +42,28 @@
 		},
 		"3": {
 			"label": "Basic Set Level",
-			"description": "0: Turn Off // 1 to 99: Dim Level // 255: On",
+			"description": "Defines the Basic Set level; 0 to disable; 1-99%, 255 for full on",
```

Comment:
255 usually means "on with last non-zero value"

Reply:
I'm not sure that is the case here:

![Screen Shot 2021-02-11 at 12 58 40 AM](https://user-images.githubusercontent.com/33612110/107606522-519f5e80-6c04-11eb-8f64-4088b5c2cb2a.png)

Reply:
It is, trust me